kafka_utils/kafka_consumer_manager/main.py

```diff
@@ -80,6 +80,11 @@ def parse_args():
         action='store_true',
         help='Show DEBUG level logging',
     )
+    parser.add_argument(
+        "--use-admin-client", action="store_true",
+        help="Use the Kafka AdminClient interface to fetch consumer "
+        "group offsets."
```

Comment:
You may consider adding a note about performance, e.g. "Use the Kafka AdminClient to fetch consumer group offsets for quicker performance"

I can imagine that many users of this CLI may not be familiar with the difference, and could benefit from a suggestion that using this is supposed to be significantly faster without any expected downsides (AFAIK, assuming the Kafka broker supports this interface)

Reply:
Done :)